classes/musicAlbum.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'item'
+
+# MusicAlbum.
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  @music_albums = []
+  def initialize(on_spotify:, **attributes)
```

Comment:
from given design we have **attribute** propertie can your replace with publish_date